NetCord/Embed.cs

```diff
@@ -1,24 +1,83 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Displays embedded content such as an image or URL, alongside a title and various other fields. All text in the embed must add up to under 6000 characters, and you can only have up to 10 embeds per message.
+/// </summary>
 public class Embed : IJsonModel<JsonModels.JsonEmbed>
 {
     JsonModels.JsonEmbed IJsonModel<JsonModels.JsonEmbed>.JsonModel => _jsonModel;
     private readonly JsonModels.JsonEmbed _jsonModel;
-
+    
+    /// <summary>
+    /// The text that is placed above the description, usually highlighted. Also directs to a URL if given, has a 256 character limit.
+    /// </summary>
+    
     public string? Title => _jsonModel.Title;
+    
+    /// <summary>
+    /// The type of the embed, always set to <see cref="EmbedType.Rich"> for webhook embeds. </see>.
+    /// </summary>
     public EmbedType? Type => _jsonModel.Type;
+    
+    /// <summary>
+    /// The part of the embed where most of the text is contained, limited to 4096 characters.
+    /// </summary>
     public string? Description => _jsonModel.Description;
+    
+    /// <summary>
+    /// The link to the address of the webpage. Mostly used with the thumbnail, icon and author elements in order to link to an image.
+    /// </summary>
     public string? Url => _jsonModel.Url;
+    
+    /// <summary>
+    /// Time that the embed was posted. Located next to the footer.
+    /// </summary>
     public DateTimeOffset? Timestamp => _jsonModel.Timestamp;
+    
+    /// <summary>
+    /// Color of your embed’s border, usually in hexadecimal or decimal, specified by an integer between 0x000000 and 0xFFFFFF.
```

Comment:
Its value is wrapped by `NetCord.Color`, I don't think we need the description about the representation and range